prism-highlighter.html

```diff
@@ -59,7 +59,12 @@
         return;
       }
 
+      if(event.detail._highlighted === true) {
```

Comment:
I don't think it's very likely. I think it's best I change this to use stopPropagation. Probably was just over thinking this.